Modify rule S3923: LaYC identical branches (#2049)
SonarSource/rspec: rules/S3923/abap/rule.adoc

== Why is this an issue?

> Having all branches in a `CASE` or `IF` chain with the same implementation is an error. Either a copy-paste error was made and something different should be executed, or there shouldn't be a `CASE`/`IF` chain at all.

Reviewer: ```suggestion
Having all branches in a `CASE` or `IF` chain with the same implementation is odd. Either a copy-paste error was made, and something different should be executed, or there shouldn't be a `CASE`/`IF` chain.
```
To be less harsh?
(+Grammarly)
(and same comment for the other languages)

Author: done

> [source,abap]

Reviewer: Could you find a sentence to introduce this code?
(and same comment for the other languages)

Author: done
